### ISSUE TITLE
--resources-log and --clear-logs fail in Python 3[.7]

### ISSUE THREAD

**Issue by skelliest** [bigmlcom/bigmler#234]
To reproduce:

1. Install `bigmler` for Python3
2. Run

```
bigmler execute \
                --code '(+ 1 1)' \
                --output-dir tmp \
                --resources-log foo
```
or

```
bigmler execute \
                --code '(+ 1 1)' \
                --output-dir tmp \
                --clear-logs
```

Expected behavior:
Both succeed, and the first command writes to the log file foo

Actual behavior:
Fails with stack trace

```
Traceback (most recent call last):
  File "/usr/local/bin/bigmler", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/bigmler/bigmler.py", line 97, in main
    bd.subcommand_dispatcher(subcommand, new_args)
  File "/usr/local/lib/python3.7/site-packages/bigmler/dispatchers.py", line 55, in subcommand_dispatcher
    return globals()["%s_dispatcher" % subcommand.replace("-", "_")](args)
  File "/usr/local/lib/python3.7/site-packages/bigmler/execute/dispatcher.py", line 63, in execute_dispatcher
    execute_whizzml(command_args, api, session_file)
  File "/usr/local/lib/python3.7/site-packages/bigmler/execute/dispatcher.py", line 78, in execute_whizzml
    clear_log_files([log])
  File "/usr/local/lib/python3.7/site-packages/bigmler/dispatcher.py", line 105, in clear_log_files
    open(log_file, 'w', 0).close()
ValueError: can't have unbuffered text I/O
```

Suggested fix:

This line https://github.com/bigmlcom/bigmler/blob/919c0b7bdcac73b8d6135978aec43135fe321624/bigmler/dispatcher.py#L105

```
open(log_file, 'w', 0).close()
```

This works in Python2 but not Python3. Internet seems to suggest replacing with

```
open(log_file, 'wb', 0).close()
```

Which works in both (tested on Python 2.7.16)

**Comment by mmerce** [comment]
Thanks for your contribution! The issue should be fixed in #236

**Comment by skelliest** [comment]
@mmerce Amazing, works for me. Thanks so much.
